includes/CheckFor.inc

```diff
@@ -208,4 +208,61 @@ class CheckFor {
     }
     return TRUE;
   }
+
+  /**
+   * Check file for redirects.
+   *
+   * @param object $row
+   *   A row object as delivered by migrate.
+   *
+   * @return bool
```

Comment:
This should return FALSE if none is found, but  if it finds one it should return the full uri of the redirect.